drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -741,30 +741,38 @@ def _get_zone_by_uid(self, uid):
     def _get_item_stats(self):
         """
         Returns a tuple representing the number of correctly-placed items,
-        and the total number of items that must be placed on the board (non-decoy items).
+        and the total number of items on the board (including decoy items).
```

Comment:
Decoy items are supposed to be left in the bank (not placed on the board), so I think it would be better if the above line said: "and the total number of required items (including decoy items)."